### DISCUSSION TITLE
Development Discussions

### DISCUSSION THREAD

**Discussion by bal-e** [beetbox/beets#5340] [General]
Hi, I'm looking to put more effort into contributing to `beets` over the next few months.  While I have a lot of big ideas for changes to make, I'm looking for a good place to discuss them with the current maintainers before I start implementing anything.  Is there an ideal place to have those discussions: something better than the General category here in GitHub Discussions or just opening a new issue for each one?  I'm talking about things like potential API changes, major PRs (like the recent `pathlib` attempts) and using new tools for development.

**Comment by arsaboo** [comment]
Discussions or Issues should be fine.

**Comment by Serene-Arc** [comment]
As @arsaboo said, issues or discussions are good for proposals and the like that are fleshed out. However, if you want something a little more informal, we also have a Zulip chat for maintainers! That way you can coordinate with us and broach collaborations or big changes. If you'd like, I can send an invite to an email of your preference, or give another channel of communication if you don't want to share your email here.

**Reply by bal-e** [reply]
That would be awesome!  You can check my recent commits to this repository for an e-mail address.  Thanks!

**Reply by Serene-Arc** [reply]
Invite sent :)
